API/Controllers/UserController.cs

```diff
@@ -39,15 +40,17 @@ public class UserController : ControllerBase
 
         private readonly IMapper mapper;
         private readonly IUserService userService;
+        private readonly IProjectService projectService;
 
         /// <summary>
         ///     Initialize a new instance of UserController
         /// </summary>
         /// <param name="userService"></param>
         /// <param name="mapper"></param>
```

Comment:
Parameter 'projectService' has no matching param tag in the XML comment for 'UserController.UserController(IUserService, IProjectService, IMapper)' (but other parameters do)